src/main/java/com/hillel/items_exchange/dto/AdvertisementDisplayDto.java

```diff
@@ -0,0 +1,46 @@
+package com.hillel.items_exchange.dto;
+
+import java.util.List;
+
+import javax.validation.constraints.Positive;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import com.hillel.items_exchange.mapper.transfer.Exist;
+import com.hillel.items_exchange.model.enums.DealType;
+import com.hillel.items_exchange.model.enums.Gender;
+import com.hillel.items_exchange.model.enums.Season;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode(exclude = {"advertisementId", "ownerAvatar", "category", "location", "images"})
```

Comment:
Do we really use equals and hashcode for that dto?

Reply:
Removed